Gemfile

```diff
@@ -9,3 +9,4 @@ gem "rubocop", "0.51.0", require: false
 gem "scss_lint", "0.53.0", require: false
 gem "sqlint", require: false
 gem "travis", "1.8.8", require: false
+gem "brakeman", :require => false
```

Comment:
I dont know much about the Gemfile but the line you have added seems a bit different than the others ;P
That is the `:require => false` is in different format.

Reply:
https://brakemanscanner.org/docs/install/
```gem "brakeman", :require => false```